.github/CODEOWNERS

```diff
@@ -10,4 +10,4 @@
 # and another the rest of the directory.
 
 /.*                                     @DataDog/agent-e2e-testing
-
+/aws/scenarios/microVMs/                @DataDog/ebpf-platform
```

Comment:
Add both `@DataDog/agent-e2e-testing` and `@DataDog/ebpf-platform`

Reply:
Done